.github/scripts/migrate_auxiliary_files.py

```
@@ -0,0 +1,416 @@
+
+
+def verify(backend: str) -> int:
+    """Fetch every registered auxiliaryFilesUrl anonymously and report the status.
+
+    This is the check the rule asks for: no credentials, no billing project —
+    exactly what a site visitor gets.
+    """
+    urls = sorted(registered_urls(backend))
+    bad = 0
+    for url in urls:
+        request = urllib.request.Request(url, method="HEAD")
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)